Python/problems/_0001_two_sum.py

```diff
@@ -17,9 +17,3 @@ def two_sum(cls, nums: List[int], target: int) -> List[int]:
             except KeyError:
                 known_items[item] = index
```

Comment:
Você precisa renomear o arquivo dessa maneira?

Reply:
Eu optei por colocar o underscore no começo, mas a questão é que não pode começar com número por ser um identificador inválido.
https://www.askpython.com/python/python-identifiers-rules-best-practices

Uma outra opção é mudar a ordem e renomear o arquivo como **two_sum_0001** mas vai fugir do padrão do projeto.
O que você acha melhor?

Reply:
Sem problema, só queria entender o unoderscore no início mesmo, pode deixar ele, os arquivos em Python podem ser assim, só da o merge Raulzinho